Makefile

```diff
@@ -184,6 +184,10 @@ unit: $(BTCD_BIN)
 	@$(call print, "Running unit tests.")
 	$(UNIT)
 
+unit-module:
+	@$(call print, "Running submodule unit tests.")
+	scripts/unit_test_modules.sh
```

Comment:
Maybe we'll even always want to run this, so add the call to this script to the already existing `unit` goal?

Reply:
I kinda like the current `unit` goal tho, as I use `make unit log="stdlog trace" pkg=$pkg case=$case` a lot to check existing tests. If we wanna combine `unit-module` and `unit`, I think we'll need more flags to run a specific test case? That being said, I also wanna do `make unit-module pkg=$pkg case=$case` someday, guess it's something we can add in a follow-up PR.